alphalens/performance.py

```diff
@@ -15,7 +15,9 @@
 
 import pandas as pd
 import numpy as np
+import warnings
 
+from pandas.tseries.offsets import Day
```

Comment:
Should that be `BDay`?

Reply:
I was in doubt. That is used only to save an error condition that should never happen (freq not set in the factor_data) so it's just a "safety belt" default value. I can switch that to BDay though